src/ykfde-enroll

```diff
@@ -40,6 +43,14 @@ while getopts ":d:s:cvh" opt; do
       CHANGE_KEYSLOT=1
       echo "INFO: Changing existing LUKS keyslot"
       ;;
+    k)
+      YKFDE_SLOT_KILL=1
+      echo "INFO: Killing existing LUKS keyslot"
```

Comment:
As this is potentially dangerous action, could you change it to something like:
`      echo "WARNING: Killing existing LUKS keyslot. If it's the last configured keyslot, the device will be inaccessible"`